lib/tidewave/mcp/server.ex

```diff
@@ -173,8 +173,9 @@ defmodule Tidewave.MCP.Server do
       is_nil(client_version) ->
         {:error, "Protocol version is required"}
 
-      client_version != unquote(@protocol_version) ->
-        {:error, "Unsupported protocol version. Server supports: #{unquote(@protocol_version)}"}
+      client_version >= unquote(@protocol_version) ->
```

Comment:
The only problem is when the client requests a version that is smaller than our version, right?

The protocol says

> In the initialize request, the client MUST send a protocol version it supports. This SHOULD be the latest version supported by the client.
> 
> If the server supports the requested protocol version, it MUST respond with the same version. Otherwise, the server MUST respond with another protocol version it supports. This SHOULD be the latest version supported by the server.
> 
> If the client does not support the version in the server’s response, it SHOULD disconnect.

So as long as the client version is >= our version, we are good

Reply:
but we need to properly parse the version to compare it

Reply:
nevermind, the string comparison probably just works with the date format?